### PR TITLE
Mimir 339 statistics job log

### PR DESCRIPTION
**Pull request by Garlov**
MIMIR-339

### CODE REVIEW

File: src/main/resources/react4xp/_entries/Dashboard/containers/Statistics/index.jsx [hunk 1]
```diff
@@ -11,13 +11,12 @@ import { selectContentStudioBaseUrl, selectInternalBaseUrl } from '../HomePage/s
 import { setOpenStatistic, setOpenModal } from './actions'
 
 import { StatisticsLog } from './StatisticsLog'
-import { selectLoading as selectQueryLoading } from '../DataQueries/selectors'
 import { RefreshStatisticsModal } from '../../components/RefreshStatisticsModal'
 
 export function Statistics() {
   const statistics = useSelector(selectStatisticsWithRelease)
+  console.log(statistics)
```
Comment (itemodahl) [inline src/main/resources/react4xp/_entries/Dashboard/containers/Statistics/index.jsx:18]: Logging

File: src/main/resources/lib/ssb/statistic.ts [hunk 2]
```diff
@@ -254,6 +280,59 @@ function prepStatistics(statistics: Array<Content<Statistics>>): Array<Statistic
...
+//     log.info(`_path LIKE "/queries/${id}/*" AND data.by.login = "${userLogin}" AND range("_ts", instant("${from}"), instant("${to}"))`)
+//     const eventLogsResult: NodeQueryResponse = connection.query({
+//       query: `_path LIKE "/queries/${id}/*" AND data.by.login = "${userLogin}" AND range("_ts", instant("${from}"), instant("${to}"))`,
+//       count: 10,
+//       sort: '_ts DESC'
+//     })
+//     return null
+//   })
+// }
+
```
Comment (itemodahl) [inline src/main/resources/lib/ssb/statistic.ts:335]: Commented code

**Review by itemodahl — APPROVED** [review]
Very nice. Just remove commented code and a log.

File: src/main/resources/react4xp/_entries/Dashboard/containers/Statistics/StatisticsLog.jsx [hunk 3]
```diff
@@ -37,30 +25,39 @@ export function StatisticsLog(props) {
   }
```
Comment (itemodahl) [inline src/main/resources/react4xp/_entries/Dashboard/containers/Statistics/StatisticsLog.jsx:25]: openEventlog not in use

File: src/main/resources/react4xp/_entries/Dashboard/containers/Statistics/StatisticsLog.jsx [hunk 4]
```diff
@@ -37,30 +25,39 @@ export function StatisticsLog(props) {
...
-        }
-      }
+    if (statistic.logData.length > 0) {
+      const log = statistic.logData[0]
+      const groupedDataSourceLogs = groupBy((log) => {
+        return log.status
+      })(log.result)
+      return (
+        <React.Fragment>
+          <span className="d-sm-flex justify-content-center text-center small haveList" onClick={() => {} /* openEventlog()*/}>
```
Comment (itemodahl) [inline src/main/resources/react4xp/_entries/Dashboard/containers/Statistics/StatisticsLog.jsx:35]: Commented code

**Review by itemodahl — APPROVED** [review]

File: src/main/resources/react4xp/_entries/Dashboard/containers/Statistics/StatisticsLog.jsx (hunk 4 above)
Comment (Garlov) [reply src/main/resources/react4xp/_entries/Dashboard/containers/Statistics/StatisticsLog.jsx]: fixed

File: src/main/resources/react4xp/_entries/Dashboard/containers/Statistics/StatisticsLog.jsx (hunk 3 above)
Comment (Garlov) [reply src/main/resources/react4xp/_entries/Dashboard/containers/Statistics/StatisticsLog.jsx:26]: fixed

File: src/main/resources/lib/ssb/statistic.ts (hunk 2 above)
Comment (Garlov) [reply src/main/resources/lib/ssb/statistic.ts:336]: added note, this is an example on how to get eventlogs for datasource refresh on statistics job log

File: src/main/resources/react4xp/_entries/Dashboard/containers/Statistics/index.jsx (hunk 1 above)
Comment (Garlov) [reply src/main/resources/react4xp/_entries/Dashboard/containers/Statistics/index.jsx]: removed

**Pull request merged**